tutorials/beginner/logs.md

```diff
@@ -6,11 +6,17 @@ layout: tutorial-beginner
 
 # Looking to the logs
 
-While using openHAB, we will need to look the the logs generated by the software.
+While using openHAB, we will need to look at the the logs generated by the software to see if there are any issues that need your attention.
```

Comment:
`we will need` and `need your attention` doesn't match.

I would suggest to change the first part to `you will need`.